homeassistant/components/islamic_prayer_times/sensor.py

```diff
@@ -45,10 +45,8 @@ def unique_id(self):
     @property
     def native_value(self):
         """Return the state of the sensor."""
-        return (
-            self.client.prayer_times_info.get(self.sensor_type)
-            .astimezone(dt_util.UTC)
-            .isoformat()
+        return self.client.prayer_times_info.get(self.sensor_type).astimezone(
```

Comment:
Is the timeZone conversion still needed here?

Reply:
Yes. I had initially removed it but then it has no timezone and the test failed https://github.com/home-assistant/core/runs/4453399367?check_suite_focus=true#step:11:762

It's because the cloud API that powers this integration only returns the times, and it's combined with the current date and passed to `dt_util.parse_datetime` here: https://github.com/home-assistant/core/blob/dev/homeassistant/components/islamic_prayer_times/__init__.py#L167-L169